src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -37,6 +37,11 @@ export default function CSVFileImport({ url, title }: CSVFileImportProps) {
             params: {
                 name: encodeURIComponent(file.name),
             },
+            headers: {
+                Authorization: `Basic ${localStorage.getItem(
```

Comment:
I guess that we should add Authorization header in case we have something in the local storage.
I mean that Authorization: `Basic null` header is not valid at all and we should not provide it at all.

Reply:
Ye, seems reasonable. Changed, thanks